acceptance/Rakefile

```diff
@@ -18,16 +18,11 @@ module HarnessOptions
     :type => 'aio',
     :helper => ['lib/helper.rb'],
     :tests  => ['tests'],
-    :log_level => 'info',
-    :color => false,
```

Comment:
i don't believe so.  we have color turned on in other jenkins jobs.  we turn color output off in some clients under tests (eg: orch-client) so we can easily parse the output